<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCardDialog</class>
 <widget class="QDialog" name="AddCardDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Card</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
    <item>
      <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
          <widget class="QLabel" name="labelOriginal">
            <property name="text"><string>Original:</string></property>
          </widget>
        </item>
        <item row="0" column="1">
          <widget class="QLineEdit" name="editOriginal"/>
        </item>
        <item row="1" column="0">
          <widget class="QLabel" name="labelTranslation">
            <property name="text"><string>Translation:</string></property>
          </widget>
        </item>
        <item row="1" column="1">
          <widget class="QLineEdit" name="editTranslation"/>
        </item>
      </layout>
    </item>
    <item>
      <layout class="QHBoxLayout" name="buttonLayout">
        <item>
          <spacer name="horizontalSpacer">
            <property name="orientation">
              <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint">
              <size><width>40</width><height>20</height></size>
            </property>
          </spacer>
        </item>
        <item>
          <widget class="QPushButton" name="btnCancel">
            <property name="text"><string>Cancel</string></property>
          </widget>
        </item>
        <item>
          <widget class="QPushButton" name="btnAdd">
            <property name="text"><string>Add</string></property>
          </widget>
        </item>
      </layout>
    </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
